source/puppet/4.9/style_guide.markdown

```diff
@@ -412,10 +412,10 @@ file {
     * => $defaults ;
 
   '/tmp/motd' :
-   content => "message of the day" ;
+   content => 'message of the day' ;
```

Comment:
I agree I don't like the space before it either, it just looks wrong, I'll submit a second cleanup in a day or two.